examples/weaviate/personalized_description_with_weaviate_and_gemini_api.ipynb

```diff
@@ -0,0 +1,963 @@
+{
```

Comment:
<p>Fortunately you don't need to use separate models for modalities any more. Default to <code>gemini-1.5-flash-latest</code> and you should get great results across text/image/audio/etc. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/76/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Reply:
<p>Awesome point! Since we're connecting to the API outside of Weaviate, I'll switch this to the flash model.</p>